bindings/python/examples/how_tos/native_tokens/create.py

```diff
@@ -44,7 +44,7 @@
     utf8_to_hex('Hello, World!'),
 )
 
-prepared_transaction = account.prepare_create_native_token(params, None)
+prepared_transaction = account.create_native_token(params, None)
 transaction = prepared_transaction.send()
```

Comment:
Do we still need this?

Reply:
Yes, because prepare_create_native_token() shouldn't be changed, undid this